src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -3279,22 +3287,34 @@ public int GetHashCode([DisallowNull] SemanticEditInfo obj)
             ISymbol newSymbol,
             SyntaxNode? newNode,
             Compilation newCompilation,
+            EditAndContinueCapabilities capabilities,
             out bool hasGeneratedAttributeChange,
             out bool hasGeneratedReturnTypeAttributeChange,
+            out bool hasParameterRename,
             CancellationToken cancellationToken)
         {
             var rudeEdit = RudeEditKind.None;
 
             hasGeneratedAttributeChange = false;
             hasGeneratedReturnTypeAttributeChange = false;
+            hasParameterRename = false;
 
             if (oldSymbol.Kind != newSymbol.Kind)
             {
                 rudeEdit = (oldSymbol.Kind == SymbolKind.Field || newSymbol.Kind == SymbolKind.Field) ? RudeEditKind.FieldKindUpdate : RudeEditKind.Update;
             }
             else if (oldSymbol.Name != newSymbol.Name)
             {
-                rudeEdit = RudeEditKind.Renamed;
+                if (oldSymbol is IParameterSymbol &&
+                    newSymbol is IParameterSymbol &&
+                    capabilities.HasFlag(EditAndContinueCapabilities.UpdateParameters))
+                {
+                    hasParameterRename = true;
+                }
+                else
+                {
+                    rudeEdit = RudeEditKind.Renamed;
```

Comment:
>RudeEditKind.Renamed [](http://example.com/codeflow?start=31&length=20)

I think this should be a different rude edit that says "... not supported by runtime"